WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaInserterHelper.swift

```diff
@@ -267,7 +267,11 @@ class GutenbergMediaInserterHelper: NSObject {
                     case .failure:
                         strongSelf.gutenberg.mediaUploadUpdate(id: mediaUploadID, state: .failed, progress: 0, url: nil, serverID: nil)
                     case .success(let value):
-                        strongSelf.gutenberg.mediaUploadUpdate(id: mediaUploadID, state: .succeeded, progress: 1, url: value.videoURL, serverID: mediaServerID)
+                        var metadata: [String: Any] = [:]
+                        if media.videopressGUID != nil {
```

Comment:
I applied this suggestion in https://github.com/wordpress-mobile/WordPress-iOS/pull/20144/commits/1472de844061dbe0a1875c20d3c623f1dfbb888d.

Reply:
@twstokes I tested the PR with this suggestion and works as expected. Since the change shouldn't modify the logic and the PR is already approved, I'll go ahead and merge it. Please let me know if you have any concerns about merging without your approval, I'll be happy to revert the changes if needed. Thanks!

Reply:
LGTM to me @fluiddot - sorry for the delayed response!

Reply:
> LGTM to me @fluiddot - sorry for the delayed response!

Great, thanks 🙇 ! 

> Feel free to merge whenever you're ready (I understand if we're wanting to wait for the non-required checks to get sorted before we do).

Yep, I paused the merge until all CI jobs pass. By the time I'm posting this comment all of them are 🟢 , so I'll proceed with merging the PR.